doc/articles/get-started-dotnet-new.md

```diff
@@ -108,4 +108,4 @@ To use it:
 
 ### Getting Help
 
-If you need help with Uno Platform please visit our [Discord](https://wwww.platform.uno/discord) - #uno-platform channel or [StackOverflow](https://stackoverflow.com/questions/tagged/uno-platform) where our engineering team and community will be able to help you. 
+If you need help with Uno Platform please visit our [Discord](https://www.platform.uno/discord) - #uno-platform channel or [StackOverflow](https://stackoverflow.com/questions/tagged/uno-platform) where our engineering team and community will be able to help you. 
```

Comment:
I think we can simply remove the `www.` in the url of the website.

@sasakrsmanovic ?

Reply:
We could, but I would dispute 'simply', because it'd require me to change the PR 😋